Patch.py

```diff
@@ -299,141 +179,3 @@ def read_rom(stream, strip_header=True) -> bytearray:
     if strip_header and len(buffer) % 0x400 == 0x200:
         return buffer[0x200:]
     return buffer
-
-
-if __name__ == "__main__":
```

Comment:
do we really want to get rid of `python Patch.py path/to/patch` completely? I used that a couple of times during development. A stripped-down version is probably enough though.

Reply:
A wrapper around APDeltaPatch could have some use, though a cmd flag for the client to behave as just a patcher could accomplish the same thing.